README.md

```diff
@@ -0,0 +1,2 @@
+# object_recognition_msgs
+### This fork is part of Acutronic Robotics' efforts in the migration of Moveit! to ROS 2.0
```

Comment:
Could you update this comment, since this is the main object_recognition_msgs repo, and not a fork?

Reply:
Of course! I just removed the added README as the other branches don't have one either.